lib/queue_classic_plus.rb

```diff
@@ -15,14 +15,18 @@ module QueueClassicPlus
 
   def self.migrate(c = QC::default_conn_adapter.connection)
     conn = QC::ConnAdapter.new(connection: c)
-    conn.execute("ALTER TABLE queue_classic_jobs ADD COLUMN last_error TEXT")
-    conn.execute("ALTER TABLE queue_classic_jobs ADD COLUMN remaining_retries INTEGER")
+    conn.execute("ALTER TABLE queue_classic_jobs ADD COLUMN IF NOT EXISTS last_error TEXT")
+    conn.execute("ALTER TABLE queue_classic_jobs ADD COLUMN IF NOT EXISTS remaining_retries INTEGER")
+    conn.execute("ALTER TABLE queue_classic_jobs ADD COLUMN IF NOT EXISTS lock BOOLEAN NOT NULL DEFAULT FALSE")
```

Comment:
Does creating a column with default values lock the table whilst the column is being added? I can't remember if this is one of those things that causes issues? From the Strong Migrations gem it looks like it's not a problem for pgsql >11 ?

https://github.com/ankane/strong_migrations#adding-a-column-with-a-default-value

Reply:
Yeah, seems ok. And I'm planning to ship during a very low traffic time when there is hardly anything in the queue.